src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -1017,6 +1017,154 @@ RppStatus rppt_slice_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             Rpp32s pixelationIntensity,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    RppLayoutParams srcLayoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+    RpptDescPtr interDescPtr = srcDescPtr;
+    RpptImagePatchPtr internalDstImgSizes = (RpptImagePatch *) calloc(dstDescPtr->n, sizeof(RpptImagePatch));
```

Comment:
Please use static_cast instead of c style casting
Applicable to all such new instances added

Reply:
@HazarathKumarM Is this addressed?

Reply:
Done